scripts/util.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/python
+
+import json
+import time
+import elasticsearch
+
+class Logger:
+  MAIN_LOG_FORMAT_STR='%(asctime)s.%(msecs)03d %(levelname)s:  %(message)s'
```

Comment:
It looks like you tab with two spaces here but four in other files. Try to keep it consistent, but that can be done in the same PR you make the other file (that Shweta mentioned) a class.